### PR TITLE
Bump origin spec files to be consistent with shared code

### PR DESCRIPTION
**Pull request by smarterclayton**
Same code from new shared repo.  Also switches to use rpmbuild directly instead of tito and does direct reuse of binary output.  openshift/release#361

### CODE REVIEW

**Comment by smarterclayton** [comment]
@stevekuznetsov I’m going to get this after we cut the branch for stage, and then cut rc.1.  Please take a pass on my changes, but this is consistent with the release PR

**Comment by smarterclayton** [comment]
This cuts another 3 minutes off all rpm builds

File: hack/lib/build/binaries.sh [hunk 1]
```diff
@@ -187,16 +190,25 @@ os::build::internal::build_binaries() {
 
     os::build::export_targets "$@"
 
+    if [[ ! "${targets[@]:+${targets[@]}}" || ! "${binaries[@]:+${binaries[@]}}" ]]; then
```
Comment (stevekuznetsov) [inline hack/lib/build/binaries.sh]: `-z` over `!` ?

Comment (smarterclayton) [reply hack/lib/build/binaries.sh]: i was having some problem with it here - will try to standardize in a follow up

**Comment by smarterclayton** [comment]
Branch is cut, merging

**Comment by smarterclayton** [comment]
Jumping the queue for origin release

**Pull request merged**